posts/2021-10-05-configurable-response-headers-21.0.0.11-beta.adoc

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title: "Configurable Response Headers and more exciting new features in Open Liberty 21.0.0.11-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Configurable Response Headers and more exciting new features in Open Liberty 21.0.0.11-beta - OpenLiberty.io
+seo-description: Open Liberty 21.0.0.11-beta provides new configurable response headers, offering more granular control over response headers, as well as important changes to the Jakarta EE convenience features, with the removal of Jakarta EE 9.0 features in the transition to Jakarta EE 9.1.
+blog_description: Open Liberty 21.0.0.11-beta provides new configurable response headers, offering more granular control over response headers, as well as important changes to the Jakarta EE convenience features, with the removal of Jakarta EE 9.0 features in the transition to Jakarta EE 9.1.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Configurable Response Headers and more exciting new features in Open Liberty 21.0.0.11-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 21.0.0.11-beta provides new configurable response headers, offering more granular control over response headers, as well as important changes to the Jakarta EE convenience features, with the removal of Jakarta EE 9.0 features in the transition to Jakarta EE 9.1.
```

Comment:
for clarity, suggesting breaking it into two sentences instead of trying to fit everything into one:

Open Liberty 21.0.0.11-beta provides new configurable response headers, which offer more granular control over response headers. This beta release also introduces important changes to the Jakarta EE convenience features, with the removal of Jakarta EE 9.0 features in the transition to Jakarta EE 9.1.